### ISSUE TITLE
XR/VR headset

### ISSUE THREAD

**Issue by kfarwell** [lucide-icons/lucide#1421]
## Icon Request

* Icon name: xr-headset
* Use case: XR/VR apps
* _Screenshots_ of similar icons:
![XR headset 1](https://github.com/lucide-icons/lucide/assets/4172356/c0b2e10d-12a2-4b82-a853-c0fbb39183c7) ![XR headset 2](https://github.com/lucide-icons/lucide/assets/4172356/1119c7fa-fccd-4687-b726-cbbb10d7ee60) ![XR headset 3](https://github.com/lucide-icons/lucide/assets/4172356/50c885b7-a370-4840-9449-bf99193cb5fa) ![XR headset 4](https://github.com/lucide-icons/lucide/assets/4172356/00bbd38b-7a3f-40a3-9d67-9f8defed4387) ![XR headset 5](https://github.com/lucide-icons/lucide/assets/4172356/36cfa821-8fe6-42e9-9abe-ec9c9d62d88c)

**Comment by jguddas** [comment]
#1344 related

**Comment by EthanHazel** [comment]
Here's a concept I came up with, how's this look?
![image](https://github.com/lucide-icons/lucide/assets/134425461/a5ed4929-b919-4e5f-8b00-8d53faa437d0)

**Comment by karsa-mistmere** [comment]
> Here's a concept I came up with, how's this look? ![image](https://private-user-images.githubusercontent.com/134425461/328635053-a5ed4929-b919-4e5f-8b00-8d53faa437d0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNTI5NzgsIm5iZiI6MTcxNTE1MjY3OCwicGF0aCI6Ii8xMzQ0MjU0NjEvMzI4NjM1MDUzLWE1ZWQ0OTI5LWI5MTktNGU1Zi04YjAwLThkNTNmYWE0MzdkMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwOFQwNzE3NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OTZhNTVkYWI3NTQ3MGQyNzViYjgzYTY4ZDgzNWEyYzdjNTA3ZjZjY2JmMTYwZjE4MjA5MDkyOWVmN2ZhNmY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Zq8Z6aShz0wL3mImt1ND5UnLxUuExjuxBDF0Rz-7_gg)

Personally, swimming mask comes to mind before VR goggle, but differentiating the two is going to be a bugger.
Other than that, the top part has a guideline violation (lack of 2px gap).

**Comment by EthanHazel** [comment]
Thanks for the feedback, this is my first contribution so I expected some issues lol, how's this look

![image](https://github.com/lucide-icons/lucide/assets/134425461/5cba9423-8f5e-4835-8334-aca189295922)

**Comment by EthanHazel** [comment]
That's true, This is my first contribution so I expected some issues lol. I took a more traditional approach for this one. How's this?

![image](https://github.com/lucide-icons/lucide/assets/134425461/cbf4f782-7cd2-4981-8227-c90adfae9a92)

**Comment by EthanHazel** [comment]
Here's a new version I made, I made a pr for it as well that has passed all tests at #2515 
![{AB976FDF-4C43-4EC6-A8A0-0878D880FCCB}](https://github.com/user-attachments/assets/ca4f33e4-0a45-4b8d-a478-81bd2fe6b0af)

**Comment by ericfennis** [comment]
Closed by #2515
